reportek/core/migrations/0007_auto_20180322_1256.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-03-22 12:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0006_auto_20180307_1213'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='envelopeoriginalfile',
+            name='restricted',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
+            model_name='envelopefile',
+            name='envelope',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='envelopefiles', to='core.Envelope'),
+        ),
+        migrations.AlterField(
+            model_name='envelopeoriginalfile',
+            name='envelope',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='envelopeoriginalfiles', to='core.Envelope'),
+        ),
+        migrations.AlterField(
```

Comment:
Because the choices on `ObligationSpec.workflow_class` are dynamic, Django will include them in every migration. I think for clarity we should strip them from non-workflow migrations.